apps/platform/src/app/components/run/customize-simulation/customize-simulation.component.ts

```diff
@@ -600,6 +612,87 @@ export class CustomizeSimulationComponent implements OnInit, OnDestroy {
     });
   }
 
+  public _createNewArchive(queryParams: ReRunQueryParams): Promise<Blob | null> {
```

Comment:
@AlexPatrie is _createNewArchive() used at all?  you introduced a new method, but not a new usage of this method.